rules-reviewed/quarkus/springboot/springboot-actuactor-to-quarkus.windup.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="springboot-metrics-to-quarkus" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <metadata>
+        <description>
+            This ruleset gives hints to migrate from SpringBoot Actuator to Quarkus Smallrye Health extension
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-javaee,3.0.0.Final"/>
+            <addon id="org.jboss.windup.rules,windup-rules-java,3.0.0.Final"/>
+            <addon id="org.jboss.windup.rules,windup-rules-xml,3.0.0.Final"/>
+        </dependencies>
+        <sourceTechnology id="springboot"/>
+        <targetTechnology id="quarkus"/>
+    </metadata>
+    <rules>
+        <rule id="springboot-actuator-to-quarkus-0100">
+            <when>
+                <dependency groupId="org.springframework.boot" artifactId="spring-boot-starter-actuator"/>
+            </when>
+            <perform>
+                <hint title="Replace the Spring Boot Actuator dependency with Quarkus Smallrye Health extension" effort="5" category-id="mandatory">
+                    <message>
+                        Replace the Spring Boot Actuator dependency with Quarkus Smallrye Health extension
```

Comment:
If you want to create a new line between the two sentences, you should either add a blank line or add 2 spaces at the end of the first line.

Reply:
Done! I did not know that "magic".